tests/AerospikeCacheTest.php

```diff
@@ -0,0 +1,334 @@
+
+        $this->aerospikeMock->method('initKey')
+            ->willReturn(['foo']);
+        $this->aerospikeMock->method('remove')
+            ->willReturn($aerospikeStatusCode);
+
+        $deleteSuccessful = $this->aerospikeCache->deleteItem('foo');
+
+        $this->assertSame($expectedDeleteToSucced, $deleteSuccessful);
+    }
+
+    public function provideStatusCodes(): array
+    {
+        return [
+            [\Aerospike::OK, true],
+            [\Aerospike::ERR_CLIENT, false],
+        ];
+    }
+
+    public function testShouldReadExistingRecordFromAerospike(): void
+    {
+        $mockedAerospikeKey = $this->createExpectedKey('foo', 'aerospike', 'cache');
+
+        $this->aerospikeMock->expects($this->once())
+            ->method('initKey')
+            ->willReturn($mockedAerospikeKey);
+
+        $this->aerospikeMock->expects($this->once())
+            ->method('getMany')
+            ->with($this->equalTo([$mockedAerospikeKey]))
+            ->willReturnCallback(function ($keys, &$records) {
+                $records[] =
+                    [
+                        'key' => $keys[0],
+                        'bins' => ['data' => 'bar'],
+                        'metadata' => ['ttl' => 1000, 'generation' => 2],
+                    ];
+
+                return \Aerospike::OK;
+            });
+
+        $item = $this->aerospikeCache->getItem('foo');
+
+        $this->assertTrue($item->isHit());
+        $this->assertSame('bar', $item->get());
+    }
+
+    public function testShouldReadNonExistingRecordFromAerospike(): void
+    {
+        $mockedAerospikeKey = $this->createExpectedKey('foo', 'aerospike', 'cache');
+
+        $this->aerospikeMock->expects($this->once())
+            ->method('initKey')
+            ->willReturn($mockedAerospikeKey);
+
+        $this->aerospikeMock->expects($this->once())
+            ->method('getMany')
+            ->with($this->equalTo([$mockedAerospikeKey]))
+            ->willReturnCallback(function ($keys, &$records) {
+                $records[] =
+                    [
+                        'key' => $keys[0],
+                        'bins' => null,
+                        'metadata' => null,
+                    ];
+
+                return \Aerospike::OK;
+            });
+
+        $item = $this->aerospikeCache->getItem('foo');
+
+        $this->assertFalse($item->isHit());
+    }
+
+    public function testShouldSaveUsingDataWrapperWithCorrectPolicy(): void
+    {
+        $reflection = new \ReflectionClass(AerospikeCache::class);
+
+        $doSaveMethod = $reflection->getMethod('doSave');
+        $doSaveMethod->setAccessible(true);
+
+        $testValue = ['foo' => 'fooValue'];
+        $testAerospikeKey = $this->createExpectedKey('foo', 'aerospike', 'cache');
+
+        $this->aerospikeMock->method('initkey')->willReturn($testAerospikeKey);
+
+        $this->aerospikeMock->expects($this->once())->method('put')->with(
```

Comment:
Could you please break this line in accordance with other similar cases?

```suggestion
        $this->aerospikeMock->expects($this->once())
            ->method('put')
            ->with(...)
```